src/pointer-analysis/value_set.h

```diff
@@ -56,6 +56,17 @@ class value_sett
     bool offset_is_set;
     bool offset_is_zero() const
     { return offset_is_set && offset.is_zero(); }
+
+    bool operator==(const objectt &other) const
+    {
+      return
+        offset_is_set==other.offset_is_set &&
```

Comment:
I'm wondering whether clang-format's suggestions should be taken into account?

Reply:
Resounding shrug from me. In order of diff hunks: either, either, either, either, no, either. The "no" is clearly worse than keeping one line one condition; happy to apply any of the others you like.

Reply:
Same here, so I'll hit the "merge" button as-is.